modules/renderer/map.js

```diff
@@ -311,7 +311,7 @@ export function rendererMap(context) {
     function resetTransform() {
         if (!transformed) return false;
 
-        surface.selectAll('.radial-menu').interrupt().remove();
+        surface.selectAll('.edit-menu').interrupt().remove();
```

Comment:
I think that is ok.. `selectAll` takes a CSS selector, so it can be written like:
```js
surface.selectAll('.radial-menu, .edit-menu').interrupt().remove();
```
We can tell people that the radial menu is deprecated and will be removed in iD v3.